hamsa_lomi/lib/app.dart

```diff
@@ -1,23 +1,12 @@
 // Flutter imports:
 import 'package:flutter/material.dart';
+import 'package:hamsa_lomi/presentation/theme/hamsa_theme.dart';
 
 class App extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      title: 'Flutter Demo',
-      theme: ThemeData(
```

Comment:
Wouldn't it be better to have the ThemeData be separated instead of creating a HapsaTheme widget? That way we can easily have a dark and light theme versions, and it would be better to switch between the two.

Reply:
```
import 'package:flutter/material.dart';

import 'colors.dart';

class CustomTheme {
  static ThemeData get lightTheme { //1
    return ThemeData( //2
      primaryColor: CustomColors.purple,
      scaffoldBackgroundColor: Colors.white,
      fontFamily: 'Montserrat', //3
      buttonTheme: ButtonThemeData( // 4
        shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(18.0)),
        buttonColor: CustomColors.lightPurple,
      )
    );
  }
}
```

Reply:
Then:
```
class MyApp extends StatelessWidget {
  const MyApp({Key key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Knight and Day',
      home: HomePage(),
      theme: CustomTheme.lightTheme,
    );
  }
}
```

Reply:
I saw this article: [theming-flutter-apps](https://www.raywenderlich.com/16628777-theming-a-flutter-app-getting-started)